configTools/src/main/scala/com/gu/janus/VerifyJanusData.scala

```diff
@@ -0,0 +1,43 @@
+package com.gu.janus
+
+import java.io.File
+
+import com.gu.janus.JanusConfig.JanusConfigurationException
+
+import scala.Console.{GREEN, RED, RESET}
+import scala.util.control.NonFatal
+
+
+object VerifyJanusData {
+  def main(args: Array[String]): Unit = {
+    args.headOption.fold {
+      Console.err.println(
+        s"""${RED}Error: Missing argument <output file>
```

Comment:
I think `output file` should instead refer to the janus data configuration input file?